pennylane/interfaces/execution.py

```diff
@@ -453,6 +457,8 @@ def execute(
             (within :meth:`Device.batch_transform`) to each tape to be executed. The default behaviour
             of the device batch transform is to expand out Hamiltonian measurements into
             constituent terms if not supported on the device.
+        use_device_jacobian_product (Optional[bool]): whether or not to use the device provided jacobian
+            product if it is available.

```

Comment:
Basically, everything in `execution.py` should be exactly the same as the autograd.py example. Just broke them up in case we wanted to go through this one first.